bin/impl/util.sh

```diff
@@ -59,3 +59,23 @@ function run_setup_script() {
   shift
   "$UNO_HOME/bin/impl/setup-$SCRIP.sh" "$@" 1>"$L_DIR/$SCRIP.stdout" 2>"$L_DIR/$SCRIP.stderr"
 }
+
+function saveConsole {
+  UNO_CONSOLE=''
+  for fd in {0..200}; do
+    rco="$(true 2>/dev/null >&${fd}; echo $?)"
+    rci="$(true 2>/dev/null <&${fd}; echo $?)"
+    [[ "${rco}${rci}" = "11" ]] && UNO_CONSOLE=${fd} && break
+  done
+
+  exec {UNO_CONSOLE}>&1
+  export UNO_CONSOLE
+}
+
+function printToConsole {
+  if [ -z "$UNO_CONSOLE" ]; then
+    echo $1
+  else
+    echo >&${UNO_CONSOLE} $1
```

Comment:
This redirect should not precede the arg. Should be before command or after all args.

Also, instead of `$1` use quoted array: `"$@"` else you only print the first arg.

Reply:
In the existing code the redirect preceded the arg for the echos.  I am not sure why it was like that, but I copied it.  Anyway, I fixed it.

Reply:
You're right. It doesn't matter. I was wrong. I don't know what I was thinking. Either way works for me. :smiley_cat: